.github/workflows/github-actions-demo.yml

```diff
@@ -4,6 +4,7 @@ on:
   push: # Trigger on push events
   pull_request: # Trigger on pull request events
     types: [opened, synchronize] # Trigger on PR opened and PR synchronize events
+  workflow_dispatch: # Allows the workflow to be manually triggered
```

Comment:
by "workflow" we mean github action tests ?
Is it manual (tests off) at the moment ? And if so why ?

Reply:
I guess with this line is now on. Or at least this was my intention.